blueprint/abac/intune-applications.md

```diff
@@ -117,7 +119,9 @@ The following Microsoft Endpoint Manager - Intune (Intune) applications can be f
   * Logo: -
 * Program
   * Install command: `%SystemRoot%\sysnative\WindowsPowerShell\v1.0\powershell.exe -executionpolicy bypass -file DefenderATP.ps1`
+    * The required `DefenderATP.ps1`  can be downloaded here [DefenderATP.ps1](/assets/files/abac/DefenderATP.txt)
   * Uninstall command: `%SystemRoot%\sysnative\WindowsPowerShell\v1.0\powershell.exe -executionpolicy bypass -file DefenderATP.ps1`
+    * The required `DefenderATP.ps1`  can be downloaded here [DefenderATP.ps1](/assets/files/abac/DefenderATP.txt)
```

Comment:
Is there a separate uninstall script or is it the same as the install script?

These two files should be grouped together similar to https://github.com/govau/desktop.gov.au/pull/69/commits/03b96c10bf7fc9f9fec51c7122613f44e2529124

Reply:
same script for install/remove currently, uninstall/revert not recommended as they are required security settings

Reply:
👍